hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/HederaMessageCallProcessor.java

```diff
@@ -68,6 +86,9 @@ public void start(final MessageFrame frame, final OperationTracer operationTrace
                 if (updater.isTokenAddress(frame.getRecipientAddress())) {
                     frame.setExceptionalHaltReason(ILLEGAL_STATE_CHANGE);
                     frame.setState(MessageFrame.State.EXCEPTIONAL_HALT);
+                } else if (updater.get(frame.getRecipientAddress()) == null
+                        && frame.getInputData().equals(Bytes.EMPTY)) {
```

Comment:
Ah, was not aware that we can have input data in this scenario --- removed the check now.

Should we add logic that will set the input data as an account memo if present, instead of always setting the hollow account memo to the default `LAZY_MEMO`?